main.rb

```diff
@@ -1,6 +1,6 @@
 require 'json'
-require_relative './classes/app'
 require './app_functionality/data_processing'
+require_relative './classes/app'
 
```

Comment:
- [ ] your App is working as expected however there is a bug when creating a rental because of the date  [here is the line](https://github.com/Doheera-kosi/School-Library-Ruby/blob/b1ee446c8f30c7462fe385fda59f203734df73da/app_functionality/app_rentals.rb#L20) where is the prob you should use string better  
<img width="904" alt="Screenshot 2023-01-10 at 16 10 35" src="https://user-images.githubusercontent.com/61790587/211588222-10bcfce5-6f82-4f0f-9459-dd25ae412947.png">

Reply:
Hi @youmari thank you for taking out time to review our project. I believe that the date written as "2020/20/29" is invalid as months should be less than 12. Attached is the screenshot of the correct rental being created. 

![image](https://user-images.githubusercontent.com/108437454/211595667-f633edd4-4812-4fcc-8659-5b69124a556a.png)

Reply:
@Sajeelzafar in this case you should inform the user how to enter the date cause there are multiple formats that will help the user to avoid this error